docs/src/main/sphinx/connector/phoenix.rst

```diff
@@ -28,6 +28,12 @@ nodes used for discovery of the HBase cluster:
 The optional paths to Hadoop resource files, such as ``hbase-site.xml`` are used
 to load custom Phoenix client connection properties.
 
+For HBase 2.x and Phoenix 5.x (5.1.0 or later) use:
```

Comment:
Add some information on what version is supported with the default connector name as well please

Reply:
Hmm... In the beginning says 4.14.1 or later then it add below that for 5.1.0 or later you need to use the phoenix-5 connector. So together it technically defines the supported versions.

In any case, happy to clarify if it's confusing - which it seems to be. Perhaps spell out the exact supported versions in the compatibility part...?

As for the two connectors... That's unfortunate, but not avoidable with Phoenix and HBase at this time. It's the same logic, same SQL, etc, just different versions. You cannot - easily - derive one from the other and keep all the tests, etc.

I vote for leaving the documentation in a single page since the behavior of the two connectors is identical. If and when the details diverge we can split the documentation as well.

Reply:
@mosabua How about this?

Reply:
Sounds good. Let's just make sure we keep an eye on this going forward.